docs/js-guidelines.md

```diff
@@ -8,17 +8,22 @@ Coding conventions serve the following purposes:
   
 - They facilitate copying, changing, and maintaining the code.  
 
-- They demonstrate React JS best practices. 
+- They demonstrate JS best practices. 
 
-## Basic Rules 
+## General JS Coding Guidelines
+
+This project follows AirBnB's JavaScript Style Guide.
```

Comment:
It might be worth also mentioning that some existing code was made before deciding to use this style guide so the all new code should conform to the style guide and when working on changes to any old code we should try fix it to match as well.